### PR TITLE
deprecate direct deployment method

### PR DESCRIPTION
**Pull request by maorfr**
### What type of PR is this?
cleanup

### What this PR does / why we need it?
remove no longer used deployment method - `Direct`

### Which Jira/Github issue(s) this PR fixes?
https://issues.redhat.com/browse/OSD-18588

### Special notes for your reviewer:
related to:
- https://github.com/openshift/managed-cluster-config/pull/1861
- https://github.com/openshift/managed-cluster-config/pull/1862

### CODE REVIEW

**Comment by bdematte** [comment]
I am generally +1 on this as this is not needed.
I would just suggest to take this opportunity to document in the README how to do the same with standard SSS so that it is clear and easy for future contributors.

**Comment by maorfr** [comment]
the way to do direct deployments is via app-interface. do you think that needs to be here?

**Comment by bdematte** [comment]
> the way to do direct deployments is via app-interface. do you think that needs to be here?

I would say yes (as I am not sure this is widely known and the first idea which would come to my mind would be a SSS in MCC). I think this would ease contribution and reduce risk of having such SSS in MCC in the future if we expect it to be in app-interface.

File: README.md [hunk 1]
```diff
@@ -23,12 +23,9 @@ add the policy as a new SelectorySyncSet.
...
 # Configuration
 
-All resources in `generated_deploy/` are bundled into a template that is used by config management to apply to target "hive" clusters.  The configuration for this supports two options for deployment.  They can be deployed in the template so they are:
+All resources in `generated_deploy/` are bundled into a template that is used by config management to apply to target "hive" clusters.  The configuration is deployed to the "hive" cluster inside a SelectorSyncSet.
 
-1. deployed directly to the "hive" cluster
-2. deployed to the "hive" cluster inside a SelectorSyncSet
-
-Direct deployment (#1) supports resources that are not synced down to OSD clusters.  SelectorSyncSet deployment (#2) supports resoures that _are_ synced down to OSD clusters.  Each are explained in detail here.  The general configuration is managed in a `config.yaml` file in each deploy directory.  Key things of note:
+SelectorSyncSet deployment supports resoures that are synced down to OSD clusters.  Each are explained in detail here.  The general configuration is managed in a `config.yaml` file in each deploy directory.  Key things of note:
```
Comment (typeid) [inline README.md:28]:
```suggestion
SelectorSyncSet deployment supports resources that are synced down to OSD clusters.  Each are explained in detail here.  The general configuration is managed in a `config.yaml` file in each deploy directory.  Key things of note:
```

**Review by typeid — APPROVED** [review]
Makes sense to me, and I see Benjamin had already +1d this as well.

**Pull request merged**